_posts/2018-10-09-proposition-j.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Proposition A
+---
+
+Proposition J is about requiring any "comprehensive rewrite" of the city land
```

Comment:
Please do use proper “curly quotes” everywhere.

Reply:
Hmm. I don’t think so.

Reply:
I just checked it out locally and they looked like smart quotes to me:

> Proposition J is about requiring any “comprehensive rewrite” of the…

I can still change it if you like. 🤷🏻‍♂️